projects/plugins/jetpack/extensions/blocks/premium-content/buttons/edit.js

```diff
@@ -12,13 +12,9 @@ const ALLOWED_BLOCKS = [
 ];
 
 function ButtonsEdit( { context, subscribeButton, setSubscribeButtonPlan } ) {
-	const planIds = context[ 'premium-content/planIds' ] || [];
-	const planId = planIds.reduce( ( acc, plan ) => {
-		if ( acc === null ) {
-			return plan;
-		}
-		return acc + '+' + plan;
-	}, null );
+	const planIds = useMemo( () => {
+		return context[ 'premium-content/planIds' ] || [];
+	}, [ context ] );
```

Comment:
I think you should move this out of `useMemo` as the operation you're doing here is not expensive and I don't know if `context` is a reliable way of checking to see if planIds have changed.

Reply:
Originally this just was a simple assignment, but one of the later useCallback/useEffect (I don't remember which) complained about the variable not working with the hook.  This solved that issue

Reply:
Okay, that makes sense. Can you use `context[ 'premium-content/planIds' ]` instead of `context`?